fl4health/clients/tabular_data_client.py

```diff
@@ -0,0 +1,115 @@
+from logging import INFO
+from pathlib import Path
+from typing import Dict, Sequence
+
+import pandas as pd
+import torch
+from flwr.common.logger import log
+from flwr.common.typing import Config, NDArray, Scalar
+
+from fl4health.clients.basic_client import BasicClient
+from fl4health.feature_alignment.constants import (
+    BINARY,
+    FEATURE_INFO,
+    FORMAT_SPECIFIED,
+    INPUT_DIMENSION,
+    ORDINAL,
+    OUTPUT_DIMENSION,
+)
+from fl4health.feature_alignment.tab_features_info_encoder import TabularFeaturesInfoEncoder
+from fl4health.feature_alignment.tab_features_preprocessor import TabularFeaturesPreprocessor
+from fl4health.parameter_exchange.full_exchanger import FullParameterExchanger
+from fl4health.utils.metrics import Metric
+
+
+class TabularDataClient(BasicClient):
+    def __init__(
+        self, data_path: Path, metrics: Sequence[Metric], device: torch.device, id_column: str, target_column: str
+    ) -> None:
+        super().__init__(data_path, metrics, device)
+        self.parameter_exchanger = FullParameterExchanger()
```

Comment:
We now have a function that you can override called `get_parameter_exchanger`. If you don't override it, by default it will provide a FullParameterExchanger(). So I think we can delete this line.